packages/twenty-front/src/modules/views/components/UpdateViewButtonGroup.tsx

```diff
@@ -89,7 +84,16 @@ export const UpdateViewButtonGroup = ({
 
   const { hasFiltersQueryParams } = useViewFromQueryParams();
 
-  const canShowButton = canPersistView && !hasFiltersQueryParams;
+  const { viewFiltersAreDifferentFromRecordFilters } =
+    useAreViewFiltersDifferentFromRecordFilters();
```

Comment:
nice!

Reply:
idea: we could also justify returning a diff (could be more versatile in the usage) and the condition would be diff.length > 0

but happy with the current hook!

Reply:
Always better to have one hook that does one precise job than creating a low-level swiss tool army hook.